TestFunctions.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include <assert.h>
+#include "TestFunctions.h"
+
+
+void testNumberToPair(int pairNumber,

```

Comment:
Nice separation of test functions. See if you can think of test-names that include their expectations. Do additional test-possibilities emerge?